static/app/gettingStartedDocs/react-native/react-native.tsx

```diff
@@ -92,253 +123,284 @@ Sentry.mobileReplayIntegration({
-  install: () => [
-    {
-      type: StepType.INSTALL,
-      description: tct(
-        'Sentry captures data by using an SDK within your application’s runtime. If you are using Expo, see [expoLink:How to Add Sentry to Your Expo Project]. This SDK works for both managed and bare projects.',
-        {expoLink: <ExternalLink href="https://docs.expo.dev/guides/using-sentry/" />}
-      ),
-      configurations: [
-        {
-          language: 'bash',
-          description: <div>{tct('Run [code:@sentry/wizard]:', {code: <code />})}</div>,
-          code: 'npx @sentry/wizard@latest -s -i reactNative',
-          additionalInfo: (
-            <Fragment>
-              <p>
-                {tct(
-                  '[wizardLink:Sentry Wizard] will patch your project accordingly, though you can [setupManuallyLink:setup manually] if you prefer.',
-                  {
-                    wizardLink: (
-                      <ExternalLink href="https://github.com/getsentry/sentry-wizard" />
-                    ),
-                    setupManuallyLink: (
-                      <ExternalLink href="https://docs.sentry.io/platforms/react-native/manual-setup/manual-setup/" />
-                    ),
-                  }
-                )}
-              </p>
-              <List symbol="bullet">
-                <ListItem>
-                  {t(
-                    'iOS Specifics: When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps.'
-                  )}
-                </ListItem>
-                <ListItem>
-                  {tct(
-                    "Android Specifics: We hook into Gradle for the source map build process. When you run [code:./gradlew] assembleRelease, source maps are automatically built and uploaded to Sentry. If you have enabled Gradle's [code:org.gradle.configureondemand] feature, you'll need a clean build, or you'll need to disable this feature to upload the source map on every build by setting [code:org.gradle.configureondemand=false] or remove it.",
-                    {
-                      code: <code />,
-                    }
-                  )}
-                </ListItem>
-              </List>
-            </Fragment>
-          ),
-        },
-      ],
-    },
-  ],
-  configure: params => [
-    {
-      type: StepType.CONFIGURE,
-      configurations: [
-        ...(params.isProfilingSelected
-          ? [
+const onboarding: OnboardingConfig<PlatformOptions> = {
+  install: params =>
+    isAutoInstall(params)
+      ? [
+          {
+            type: StepType.INSTALL,
+            description: tct(
+              'Run [code:@sentry/wizard] to automatically configure your project:',
+              {code: <code />}
+            ),
+            configurations: [
+              {
+                language: 'bash',
+                code: `npx @sentry/wizard@latest -s -i reactNative --org ${params.organization.slug} --project ${params.projectSlug}`,
```

Comment:
The `-s` is not necessary AFAIK.

Reply:
I tested the new command works as expected with RN.

Except the questions SaaS or self hosted and do you have an account? But that issue for all wizards.

Reply:
```suggestion
                code: `npx @sentry/wizard@latest -i reactNative --org ${params.organization.slug} --project ${params.projectSlug}`,
```